core/include/vcpu.h

```diff
@@ -238,6 +238,7 @@ struct vcpu_t {
 #ifdef CONFIG_HAX_EPT2
     struct mmio_fetch_cache mmio_fetch;
 #endif  // CONFIG_HAX_EPT2
+    int cr0_ts;
```

Comment:
`struct vcpu_t` stores the guest CPU state, so putting the host CR0.TS state here can cause confusion. How about moving it to `struct hstate` (`core/include/cpu.h`)? Also, since this is a Boolean value, I'd use `bool` instead of `int`.

Reply:
Done